hack/run-kube-local.sh

```diff
@@ -0,0 +1,21 @@
+set -euo pipefail
```

Comment:
can this be converted into a target in https://github.com/envoyproxy/gateway/blob/main/tools/make/kube.mk ?